src/altinn-app-frontend/src/store/index.ts

```diff
@@ -14,7 +14,7 @@ export const setupStore = (preloadedState?: PreloadedState<RootState>) => {
     middleware: (getDefaultMiddleware) => getDefaultMiddleware({
       // TODO: enable once we have cleaned up our store
       serializableCheck: false,
```

Comment:
Any idea whats missing to enable this as well (and do we have a task to fix it)?

Reply:
Not exactly sure. Will have to investigate further. I have added it as a sub-task on https://github.com/Altinn/app-frontend-react/issues/159.

I suggest we try to get this done in our next sprint.